### DISCUSSION TITLE
Github Pages custom website url stuck

### DISCUSSION THREAD

**Discussion by yasiralamriki** [community/community#22157] [Other Feature Feedback, Questions, & Ideas]
<p>Hello, I have changed my custom URL to <strong>[removed URL]</strong> (I dont remember why) and now it is stuck, yes I changed it to blank and saved, however it is stuck now, is there any way to fix this, or should I submit this as a bug?</p>

**Comment by canuckjacq** [comment]
<p>:wave: Welcome!</p>
<p>I can see where you have removed the CNAME. Can you elaborate on what you mean by “it is stuck”?</p>

**Comment by yasiralamriki** [comment]
<p>It redirects to that and I am not able to change it back.</p>

**Comment by canuckjacq** [comment, marked as answer]
<p>Your pages site is <code>https://ajl223.github.io/</code></p>
<p>When I click that link, nothing redirects. If it is on your browser, maybe try it again in an incognito window?</p>

**Comment by yasiralamriki** [comment]
<p>It works for me too now, it was still redirecting for an hour or two when I tried it</p>
